dfastmi/batch/DFastUtils.py

```diff
@@ -132,3 +133,14 @@ def _get_km_bins(km_bin: Tuple[float, float, float], type_characteristic_chainag
     km = km_bin[0] + dx + numpy.arange(lb, ub) * km_step
 
     return km
+
+def get_progloc() -> str:
```

Comment:
Perhaps return a `Path` object instead of string as we want to work with Paths instead of strings.

Reply:
Yes good idea. Will update